configure

```diff
@@ -3834,7 +3834,7 @@ _ACEOF
 
 
 # require automake 1.11 - color tests
-am__api_version='1.15'
+am__api_version='1.13'
```

Comment:
You're going to need to redo the bootstrap with the libMesh contrib version of auto tools.  We don't want an extra two hundred lines of diffs every time we switch from one developer's version to another's.